lib/netsuite/configuration.rb

```diff
@@ -216,10 +216,24 @@ def account(account = nil)
       if account
         self.account = account
       else
+        return sandbox_account if sandbox
         attributes[:account]
       end
     end
 
+    def sandbox_account=(sandbox_account)
+      attributes[:sandbox_account] = sandbox_account
+    end
+
+    def sandbox_account(sandbox_account = nil)
+      if sandbox_account
+        self.sandbox_account = sandbox_account
+      else
+        return attributes[:sandbox_account] if attributes[:sandbox_account]
+        "#{attributes[:account]}_SB1"
```

Comment:
Can we not just do this in this `else` block?  https://github.com/ShopifyFRS/russbot/blob/master/config/initializers/netsuite.rb#L11